src/test/scala/org/apache/spark/sql/delta/DeltaSourceSuite.scala

```diff
@@ -964,6 +964,71 @@ class DeltaSourceSuite extends DeltaSourceSuiteBase {
     }
   }
 
+  test("Read data using the .delta() method extended from the DataFrameReader API") {
+    withTempDir { tempDir =>
+      Seq(1, 2, 3).toDF("x").write.format("delta").save(tempDir.getCanonicalPath)
```

Comment:
Does it really matter to use `"x"` for the columns? Clean it up for future readers.

Reply:
removed the column name as requested 👍